migrations/0001_migrations.rb

```diff
@@ -38,5 +38,16 @@
       DateTime    :modified
       Boolean     :active, default: true
     end
+
+    create_table(:db_resources) do
+      primary_key :id
+      foreign_key :project_id, :projects
+      String      :name, unique: true
+      String      :type
+      String      :cluster
```

Comment:
We don't actually need a cluster field here, since dbs aren't in a cluster - but we may want to replace it with a server field to specify what server it is on (db2.o.o or mysql3.o.o, etc)